guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,21 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using LUKS during host provisioning.
+In addition, you can bind the disk to a Tang server through Clevis for decryption during boot.
```

Comment:
I think, we should 
- add one short sentence why clevis/tang is good to be used
- add these two links for more information on clevis/tang: 
  https://github.com/latchset/clevis
  https://github.com/latchset/tang